app/controllers/registrations_controller.rb

```diff
@@ -8,9 +8,9 @@ def create
 
     if @user.save
       login(params[:user][:email], params[:user][:password])
-      redirect_to root_path, notice: 'Добро пожаловать!'
+      redirect_to root_path, notice: t('flash.log_in')
     else
-      flash.now[:alert] = 'Заполните все поля!'
+      flash.now[:alert] = t('flash.fill_in_fields')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.